articles/postgresql/migrate/how-to-migrate-using-dump-and-restore.md

```diff
@@ -34,6 +34,12 @@ For example, if you have a local server and a database called **testdb** in it,
 pg_dump -Fc -v --host=localhost --username=masterlogin --dbname=testdb -f testdb.dump
 ```
 
+> [!NOTE]
+> If you have a custom schema(s) in postgres database (The default database you can connect to once your server is created) in Azure database for PostgreSQL flexible server and want to export it using pg_dump, use the (--schema=pattern) option in the pg_dump export command, because you can’t export the whole postgres database as it contains schemas owned by Azure superuser. For detailed syntax information, see [pg_dump]( https://www.postgresql.org/docs/current/app-pgdump.html). 
+>
```

Comment:
****

Reply:
Can o get help? Im sorta lost at the mo.ent. I also habe soomeone who linked me to a firbase console. My entire online life has been taken over. I'm trtimg to lean tho..

Reply:
what is the pending on this PR?

Reply:
maybe small "t" with "The default" -> "the default"